content/en/security_platform/application_security/custom_rules.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Custom Detection Rules
+kind: documentation
+further_reading:
+- link: "/security_platform/application_security/"
+  tag: "Documentation"
+  text: "Monitoring threats with Datadog Application Security"
+- link: "/security_platform/application_security/troubleshooting"
+  tag: "Documentation"
+  text: "Troubleshoot common Datadog Application Security issues"
+---
+
+## Overview
+
+Application Security comes with a set of [out-of-the-box detection rules][1] which aim to catch attack attempts and vulnerability triggers that impact your production systems.
+
+However, there are situations where you may want to customize a rule based on your environment. For example, you may want to customize a rule that catches attack attempts on a service’s route. For instance, a pre-production development route accepting SQL and returning the results. Catching SQL attempts is noisy as the route is restricted to internal developers, therefore you can customize this rule to exclude these patterns.
+
+Another example is customizing a rule to exclude an internal security scanner. Application Security detects its activity as expected. However, you may not want to be notified of it’s regularly occurring scan.
+
+In these situations, a custom detection rule can be created to exclude such events. This guide shows you how to create a custom detection rule for Application Security.
+
+## Configuration
+
+To customize an OOTB detection rule, you must first clone an existing rule. Navigate to your [Detection Rules][2] and select a rule. Scroll to the bottom of the rule and click the Clone Rule button. This now enables you to edit the existing rule.
+
+### Define AppSec query
+
+Construct an Application Security query. For example, create a query to monitor an endpoint for SQL injection attempts: `@appsec.type:sql_injection @http.url_details.path:"/debug-endpoint-executing-sql" env:production`.
```

Comment:
Sorry that's my bad, I thought the query would be used in the context of an Exclusion Filter. If it is a detection rule query, then it misses an `-`: `@appsec.type:sql_injection -@http.url_details.path:"/debug-endpoint-executing-sql" env:production`